flow/raster_cache.cc

```diff
@@ -109,7 +111,8 @@ static bool IsDisplayListWorthRasterizing(DisplayList* display_list,
 
   // TODO(abarth): We should find a better heuristic here that lets us avoid
   // wasting memory on trivial layers that are easy to re-rasterize every frame.
-  return display_list->op_count(true) > 5;
+  int complexity_score = complexity_calculator->compute(display_list);
+  return complexity_calculator->should_be_cached(complexity_score);
```

Comment:
That TODO is doomed. We should look at ways of not having to compute this on every frame, but my reorg of the raster cache stuff should provide a better framework for that.